.circleci/config.yml

```diff
@@ -18,20 +18,8 @@ jobs:
           make install-go
           source ~/.bashrc
           go mod download
-    - run:
```

Comment:
Hmm, having some trouble getting `make test-database-up` to work locally.  I figured it was related to the changes here https://github.com/hashicorp/boundary/pull/1543/files, can dig in or chat with Tim about it in the New Year

Reply:
Ah I see you added https://github.com/hashicorp/boundary/commit/269fb94885ab361aec13b112e9c63a81a0b3a5d0#diff-702fdcc3f995a7ce572c8b1f03ccf326caf9599142c8de0317e60ba7082e2600 recently so we do not need to spin up a testing db, in that case we can just leave this out.  We should probably remove the `make test-database-up` if its no longer needed for tests

Reply:
It's needed -- that option is not used here. We do need to spin up a testing DB.

Reply:
Oh sorry. I forgot that I set it to opt-in so this should do what's expected.